test/framework/e2e/splunk.go

```diff
@@ -0,0 +1,374 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"github.com/openshift/cluster-logging-operator/internal/constants"
+	"github.com/openshift/cluster-logging-operator/internal/k8shandler"
+	"github.com/openshift/cluster-logging-operator/internal/utils"
+	"github.com/openshift/cluster-logging-operator/test/helpers/oc"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"net/url"
+
+	apps "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	SplunkStandalone = "splunk-standalone"
+	Splunk           = "splunk"
+	splunkImage      = "splunk/splunk:9.0.0"
```

Comment:
lets retag this image and push it to quay so we ensure we have a copy of it

Reply:
@jcantrill on which org, and who has credentials todo it?

Reply:
looks like this is available to be updated https://quay.io/repository/openshift-logging/splunk?tab=tags